src/_pytest/unittest.py

```diff
@@ -140,8 +141,10 @@ def _make_xunit_fixture(
         def cleanup(*args):
             pass
 
+    scope_name = cast("_ScopeName", scope.value)
```

Comment:
Perhaps this can be done in Scope itself by overriding `value` (or just adding `value: _ScopeName` to the class)? (Didn't check if that's possible)

Reply:
Hmm good idea, but unfortunately it does not work. 😞